--- lttng-modules-2.1.0/instrumentation/syscalls/headers/syscalls_unknown.h	2011-09-28 00:30:28.000000000 +0200
+++ lttng-modules-2.1.0//instrumentation/syscalls/headers/syscalls_unknown.h	2014-01-23 16:40:27.605616619 +0100
@@ -38,13 +38,13 @@
  * event.
  */
 TRACE_EVENT(exit_syscall,
-	TP_PROTO(struct pt_regs *regs, long ret),
-	TP_ARGS(regs, ret),
+	TP_PROTO(struct pt_regs *regs, long ret1),
+	TP_ARGS(regs, ret1),
 	TP_STRUCT__entry(
 		__field(long, ret)
 	),
 	TP_fast_assign(
-		tp_assign(ret, ret)
+		tp_assign(ret, regs->ARM_r0)
 	),
 	TP_printk()
 )
