diff -ru busybox-1.20.2-orig/coreutils/stat.c busybox-1.20.2/coreutils/stat.c
--- busybox-1.20.2-orig/coreutils/stat.c
+++ busybox-1.20.2/coreutils/stat.c
@@ -99,9 +99,11 @@
 	if (S_ISFIFO(st->st_mode)) return "fifo";
 	if (S_ISLNK(st->st_mode))  return "symbolic link";
 	if (S_ISSOCK(st->st_mode)) return "socket";
+#ifndef ANDROID
 	if (S_TYPEISMQ(st))        return "message queue";
 	if (S_TYPEISSEM(st))       return "semaphore";
 	if (S_TYPEISSHM(st))       return "shared memory object";
+#endif
 #ifdef S_TYPEISTMO
 	if (S_TYPEISTMO(st))       return "typed memory object";
 #endif

