diff -ru busybox-1.20.2-orig/include/applets.src.h busybox-1.20.2/include/applets.src.h
--- busybox-1.20.2-orig/include/applets.src.h
+++ busybox-1.20.2/include/applets.src.h
@@ -47,17 +47,17 @@
 # define APPLET_NOFORK(name,main,l,s,name2)  MAKE_USAGE(#name, name2##_trivial_usage)
 
 #elif defined(MAKE_LINKS)
+# define APPLET(name,l,c)                    LINK BB_DIR_BIN name
+# define APPLET_ODDNAME(name,main,l,s,name2) LINK BB_DIR_BIN name
+# define APPLET_NOEXEC(name,main,l,s,name2)  LINK BB_DIR_BIN name
+# define APPLET_NOFORK(name,main,l,s,name2)  LINK BB_DIR_BIN name
-# define APPLET(name,l,c)                    LINK l name
-# define APPLET_ODDNAME(name,main,l,s,name2) LINK l name
-# define APPLET_NOEXEC(name,main,l,s,name2)  LINK l name
-# define APPLET_NOFORK(name,main,l,s,name2)  LINK l name
 
 #else
   static struct bb_applet applets[] = { /*    name, main, location, need_suid */
+# define APPLET(name,l,s)                    { #name, #name, BB_DIR_BIN, s },
+# define APPLET_ODDNAME(name,main,l,s,name2) { #name, #main, BB_DIR_BIN, s },
+# define APPLET_NOEXEC(name,main,l,s,name2)  { #name, #main, BB_DIR_BIN, s, 1 },
+# define APPLET_NOFORK(name,main,l,s,name2)  { #name, #main, BB_DIR_BIN, s, 1, 1 },
-# define APPLET(name,l,s)                    { #name, #name, l, s },
-# define APPLET_ODDNAME(name,main,l,s,name2) { #name, #main, l, s },
-# define APPLET_NOEXEC(name,main,l,s,name2)  { #name, #main, l, s, 1 },
-# define APPLET_NOFORK(name,main,l,s,name2)  { #name, #main, l, s, 1, 1 },
 #endif
 
 #if ENABLE_INSTALL_NO_USR

