--- lttng-ust-2.8.1/configure.ac	2016-11-03 15:57:17.793330926 +0100
+++ lttng-ust-2.8.1/configure.ac2	2016-11-03 17:07:31.489290619 +0100
@@ -437,9 +437,6 @@
 
 AC_CONFIG_FILES([
 	Makefile
-	doc/Makefile
-	doc/examples/Makefile
-	doc/man/Makefile
 	include/Makefile
 	include/lttng/ust-version.h
 	snprintf/Makefile
@@ -467,17 +464,6 @@
 	python-lttngust/setup.py
 	python-lttngust/lttngust/__init__.py
 	tools/Makefile
-	tests/Makefile
-	tests/ctf-types/Makefile
-	tests/hello/Makefile
-	tests/hello.cxx/Makefile
-	tests/same_line_tracepoint/Makefile
-	tests/snprintf/Makefile
-	tests/ust-elf/Makefile
-	tests/benchmark/Makefile
-	tests/utils/Makefile
-	tests/test-app-ctx/Makefile
-	tests/gcc-weak-hidden/Makefile
 	lttng-ust.pc
 ])
 
--- lttng-ust-2.8.1/Makefile.in	2016-11-10 19:02:07.150402152 +0100
+++ lttng-ust-2.8.1/Makefile.in2	2016-11-10 19:03:28.610400482 +0100
@@ -223,8 +223,7 @@
 	liblttng-ust liblttng-ust-ctl liblttng-ust-fork \
 	liblttng-ust-libc-wrapper liblttng-ust-cyg-profile \
 	liblttng-ust-python-agent tools liblttng-ust-dl \
-	liblttng-ust-java liblttng-ust-java-agent python-lttngust \
-	tests doc
+	liblttng-ust-java liblttng-ust-java-agent python-lttngust
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
 distdir = $(PACKAGE)-$(VERSION)
 top_distdir = $(distdir)
@@ -425,7 +424,7 @@
 	liblttng-ust liblttng-ust-ctl liblttng-ust-fork \
 	liblttng-ust-libc-wrapper liblttng-ust-cyg-profile \
 	liblttng-ust-python-agent tools $(am__append_1) \
-	$(am__append_2) $(am__append_3) $(am__append_4) tests doc
+	$(am__append_2) $(am__append_3) $(am__append_4)
 
 #temporarily disabled
 # liblttng-ust-malloc
